ui/analyse/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     { "path": "../chess/tsconfig.json" },
     { "path": "../common/tsconfig.json" },
     { "path": "../game/tsconfig.json" },
+    { "path": "../keyboardMove/tsconfig.json" },
```

Comment:
The tsc dependency mechanism isn't documented anywhere so I'm impressed. I think you're the first to do this.

Reply:
I won't pretend to understand it, but I will pretend to be quite good at pattern matching 😎

Reply:
it's actually straightforward. tsc doesn't care about our monorepo dependencies (package.json), so if you want correct behavior when type checking - you have to either compile everything in the correct order (bad) or express dependencies in tsconfig.json (better, but still not great).